WTWSMS/history/provinces/100 - Avranches.txt

```diff
@@ -22,8 +22,6 @@ religion = celtic_pagan
 terrain = forest
 # History
 1.1.1 = { # Viking takeover
-	remove_settlement = b_caen
-	remove_settlement = b_bayeux
```

Comment:
No, they have been removed and won't return. The mod is an overhaul, there is no need for us to support vanilla bookmarks which would be inaccurate. Those who want to experience those bookmarks can play directly on the vanilla game. 😉